packages/cli/src/commands/build.ts

```diff
@@ -314,7 +321,13 @@ export async function build({
 
       console.log(yellow('Run ' + bold(`cannon publish ${deployUrl}`) + ' to pin the partial deployment package on IPFS.'));
     } else {
-      console.log(greenBright(`Successfully built package ${bold(`${name}:${version}@${selectedPreset}`)} (${deployUrl})`));
+      console.log(
+        greenBright(
+          `Successfully built package ${bold(
+            `${name}:${version}@${selectedPreset}`
+          )} \n - Deploy Url: ${deployUrl} \n - Metadata Url: ${metaUrl}`
```

Comment:
displaying the metadata url here might cause confusion for those who dont understand what it is, having 2 ipfs urls. I think it would be best if we only display the deploy url

Reply:
Sounds good! The reason I displayed it is because I added functionality to the fetch command to include metadata. Not sure if thats the right way to go about it.